cookbook/specs/spec_add_aptos.json

```diff
@@ -455,6 +455,39 @@
                                         "expected_value": "1"
                                     }
                                 ]
+                            },
+                            {
+                                "name": "pruning",
```

Comment:
set this to warning severity and later we will make it error.
that will make sure that if we mess with the verifications providers wont go down

Reply:
done. see https://github.com/lavanet/lava/commit/048b31fa857659cd70dc4d7057a13d45466c3b5a